README.md

```diff
@@ -59,6 +59,7 @@ import 'vue-awesome/icons/flag'
 import 'vue-awesome/icons'
 
 // register component to use
+Vue.component('icon', Icon)
```

Comment:
Components are not necessarily registered like this so I left this out. Local components are not registered this way.